src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1021,12 +1022,15 @@ export class ResultsViewPageStore {
                     patientAttribute: true,
                 } as ClinicalAttribute);
             }
-            return serverAttributes
-                .concat(specialAttributes)
-                .concat(this.clinicalAttributes_profiledIn.result!)
-                .concat(
-                    this.clinicalAttributes_comparisonGroupMembership.result!
-                );
+            const ret: ExtendedClinicalAttribute[] = [];
```

Comment:
May be directly return value instead of creating new variable.
`return [... serverAttributes, ... specialAttributes, ...]`